export_all.py

```diff
@@ -0,0 +1,218 @@
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+import argparse
+import csv
+import os
+
+from time import time
+from datetime import datetime
+from web3 import Web3
+
+from ethereumetl.jobs.export_blocks_job import ExportBlocksJob
+from ethereumetl.jobs.export_token_transfers_job import ExportTokenTransfersJob
+from ethereumetl.jobs.export_receipts_job import ExportReceiptsJob
+from ethereumetl.jobs.export_contracts_job import ExportContractsJob
+from ethereumetl.jobs.export_tokens_job import ExportTokensJob
+
+from ethereumetl.jobs.exporters.blocks_and_transactions_item_exporter import blocks_and_transactions_item_exporter
+from ethereumetl.jobs.exporters.token_transfers_item_exporter import token_transfers_item_exporter
+from ethereumetl.jobs.exporters.receipts_and_logs_item_exporter import receipts_and_logs_item_exporter
+from ethereumetl.jobs.exporters.contracts_item_exporter import contracts_item_exporter
+from ethereumetl.jobs.exporters.tokens_item_exporter import tokens_item_exporter
+
+from ethereumetl.logging_utils import logging_basic_config
+from ethereumetl.providers.auto import get_provider_from_uri
+from ethereumetl.thread_local_proxy import ThreadLocalProxy
+from ethereumetl.csv_utils import set_max_field_size_limit
+from ethereumetl.file_utils import smart_open
+
+parser = argparse.ArgumentParser(description='Export all for a range of blocks.',
+                                 usage='-s <start_block> -e <end_block> -b <batch_size> -p <provider_uri> [-o <output_dir>]')
+parser.add_argument('-s', '--start-block', default=0, type=int, help='Start block')
+parser.add_argument('-e', '--end-block', required=True, type=int, help='End block')
+parser.add_argument('-b', '--batch-size', default=100, type=int, help='The number of blocks to export at a time.')
+parser.add_argument('-p', '--provider-uri', default='https://mainnet.infura.io', type=str,
+                    help='The URI of the web3 provider e.g. '
+                         'file://$HOME/Library/Ethereum/geth.ipc or https://mainnet.infura.io')
+parser.add_argument('-o', '--output-dir', required=True, type=str, help='Output directory, partitioned in Hive style.')
+parser.add_argument('-w', '--max-workers', default=5, type=int, help='The maximum number of workers.')
+
+args = parser.parse_args()
+
+def log(msg=''):
```

Comment:
This function is not needed anymore as the standard Python logging facilities can be used instead, i.e.:
```
logger = logging.getLogger(__name__)
logger.info('test')
```